### PR TITLE
OHRI-2156:Remove error 400 on ohri-namibia-working

### PR DESCRIPTION
**Pull request by Herobiam**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Summary
Error 400 appears whenever you save the various forms - ANC, L&D etc

## Screenshots

Uploading Screen Recording 2024-03-04 at 9.13.09 PM.mov…

### CODE REVIEW

File: packages/esm-ohri-pmtct-app/src/post-submission-actions/current-ptracker-action.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { PostSubmissionAction } from '@openmrs/openmrs-form-engine-lib';
 import { fetchPatientIdentifiers, saveIdentifier } from '../api/api';
 import { Patient, PatientIdentifier } from '../api/types';
 import { getConfig } from '@openmrs/esm-framework';
+import { use } from 'i18next';
```
Comment (arodidev) [inline packages/esm-ohri-pmtct-app/src/post-submission-actions/current-ptracker-action.ts:5]: Is this import being used anywhere?

File: yarn.lock [hunk 2]
```diff
@@ -3208,7 +3208,7 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@openmrs/openmrs-form-engine-lib@npm:next":
+"@openmrs/openmrs-form-engine-lib@npm:1.0.0-pre.544, @openmrs/openmrs-form-engine-lib@npm:next":
```
Comment (arodidev) [inline yarn.lock:3211]: I don't think we should update the `package.json` and `yarn.lock` file.

**Review by arodidev — COMMENTED** [review]
Thanks for this, although I am skeptical about why we are updating the `package.json` and `yarn.lock` files in this PR.

File: packages/esm-ohri-pmtct-app/src/post-submission-actions/current-ptracker-action.ts (hunk 1 above)
Comment (Herobiam) [reply packages/esm-ohri-pmtct-app/src/post-submission-actions/current-ptracker-action.ts:5]: Thanks let me remove that

Comment (Herobiam) [reply packages/esm-ohri-pmtct-app/src/post-submission-actions/current-ptracker-action.ts:5]: removed

**Review by alaboso — DISMISSED** [review]

**Review by alaboso — APPROVED** [review]

**Pull request merged**